runtime/src/main/java/org/corfudb/runtime/view/ManagementView.java

```diff
@@ -221,6 +226,86 @@ private ClusterStatus getClusterHealth(Layout layout, Set<String> peerResponsive
         return peerResponsiveNodes;
     }
 
+    /**
+     * Fetches the initial layout to fetch the cluster status.
```

Comment:
Attempts to fetch the layout with the highest epoch.

Reply:
Done.